guides/v2.1/contributor-guide/contributing_dod.md

```diff
@@ -15,12 +15,20 @@ At a high level, the criteria defined in our "definition of done" revolve around
 
 The following sections provide additional details about each of these criteria:
 
-## Functional Backward Compatibility
+## Backward Compatibility 
+
+### Functional Backward Compatibility
 
 Existing product features and functionality must be retained during any changes to the code.
-If backward-incompatible functional changes are made, the documentation should explain the justification and provide the "business value".
+Any backward-incompatible functional changes must be approbved by a product owner.
+The documentation should explain the justification and provide the "business value".
+
+### Technical Backward Compatibility
```

Comment:
We need to define the difference between functional and technical backwards compatibility.